src/com/apeshko/javacore/task_01/model/exception/FlightException.java

```diff
@@ -0,0 +1,7 @@
+package com.apeshko.javacore.task_01.model.exception;
+
+public class FlightException extends RuntimeException {
+    public FlightException(String errorMessage) {
+        super(errorMessage);
+    }
```

Comment:
for exception handling, don't forget to pass as a parameter original throwable to your custom exception, otherwise you won't be able to understand what exactly the issue, you have only message you provide
there is another superclass constructor for this purpose